orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DetermineRollbackCandidatesTask.java

```diff
@@ -107,8 +111,10 @@ public long getTimeout() {
   }
 
   public long getDynamicTimeout(StageExecution stage) {
-    if (stage.getContext().containsKey("rollbackTimeout")) {
-      return TimeUnit.MINUTES.toMillis((int) stage.getContext().get("rollbackTimeout"));
+    if (stageRollbackTimeout) {
```

Comment:
To improve expressiveness and simplicity, we could do something like:
``` java
var shouldUseStageRollbackTimeout = stageRollbackTimeout
 && stage.getContext().containsKey("rollbackTimeout");

if (shouldUseStageRollbackTimeout){
    return TimeUnit.MINUTES....
}
return getTimeout();
```

It's a matter of taste though, so feel free to ignore this comment.